mos-platform/rp6502/exit.c

```diff
@@ -0,0 +1,7 @@
+#include "rp6502.h"
+
+void _exit(int status) {
```

Comment:
You'll need to sync to get past the breaking commit d2313256667c80986627440bf42d918b5d7f7f68, which removes POSIX `_exit` in favor of the C standard's `_Exit`.

Reply:
done.